modules/ROOT/pages/graph-analytics/index.adoc

```diff
@@ -19,19 +22,36 @@ The Graph Analytics plugin allows you to use the link:https://neo4j.com/docs/gra
 * 4 GB of memory or more
 * Any supported cloud providers and regions
 
-You can enable or disable the plugin when necessary.
+You can enable or disable Graph Analytics when necessary.
+The plugin shares compute and memory resources with the AuraDB server, so you do not incur any additional costs when you enable it.
+
+[[aura-gds-serverless]]
+== Graph Analytics Serverless
+
+include::partial$EAP.adoc[]
+
+Graph Analytics Serverless allows you to use the Graph Data Science library regardless of where your source data is stored.
+
+Graph Analytics Serverless runs in Aura as a link:https://neo4j.com/docs/graph-data-science-client/current/gds-session/[dedicated service] optimised for analytics workloads, with no memory or compute resources shared with your data store.
```

Comment:
Should we maybe link to the server manual here? E.g. the page I'm currently writing